packages/curl/project.bri

```diff
@@ -2,12 +2,12 @@ import * as std from "std";
 import brotli from "brotli";
 import libpsl from "libpsl";
 import libssh2 from "libssh2";
-import nushell from "nushell";
 import openssl from "openssl";
 
 export const project = {
   name: "curl",
   version: "8.14.1",
+  repository: "https://github.com/curl/curl.git",
 };
 
 const source = Brioche.download(
```

Comment:
To see if later we should update the source of truth for this package since we now have the field `repository` in the `project`.